spec/models/reservation_spec.rb

```diff
@@ -1583,4 +1583,34 @@
       it { is_expected.to be_blank }
     end
   end
+
+  describe "#can_switch_instrument_off?" do
+    # let!(:account) { FactoryBot.create(:nufs_account, account_users_attributes: account_users_attributes_hash(user: user)) }
```

Comment:
Can we remove this commented out code?

Reply:
What commented out code? _whistles, looks away in embarrassment_ 🤦‍♂️